source/renderer/app/components/staking/stake-pools/StakePool.scss

```diff
@@ -0,0 +1,67 @@
+.component {
+  background-color: var(--theme-staking-stake-pool-background-color);
+  border-radius: 4px;
+  border: solid 1px var(--theme-staking-stake-pool-border-color);
+  height: 71px;
+  margin: 5px;
+  position: relative;
+  width: 80px;
+}
+
+.content {
+  height: 100%;
+  cursor: pointer;
+}
+
+.isSelected {
+  box-shadow: 0 0 4px 2px var(--theme-staking-stake-pool-glow-color);
+}
+
+.id {
+  color: var(--theme-staking-stake-pool-id-color);
+  font-size: 14px;
+  font-weight: 600;
+  letter-spacing: -0.5px;
+  line-height: 1;
+  margin: 12px 0 9px;
+  text-align: center;
+}
+
+.index {
+  font-size: 20px;
+  font-weight: bold;
+  text-align: center;
+}
+
+.clock {
+  background: var(--theme-staking-stake-pool-retirement-background-color);
+  border-radius: 0 4px;
+  height: 18px;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 18px;
+}
+
+.clockIcon {
+  display: block;
+  line-height: 0px;
+  margin: 0;
+  padding-top: 1px;
```

Comment:
@daniloprates here you set top padding to 1px and then in the following line back to zero - is that on purpose?